services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_abc.py

```diff
@@ -1,8 +1,10 @@
+# pylint: disable=relative-beyond-top-level
```

Comment:
is pylint getting it wrong??

Reply:
Might be a bug with pylint, from what I can tell.
It also sounds very wired to me!

https://github.com/flycheck/flycheck/issues/1758

Reply:
From pylint' source code, this is a bit misleading.

```
:relative-beyond-top-level (E0402): *Attempted relative import beyond top-level package*
  Used when a relative import tries to access too many levels in the current
  package.
```
[Find above here](https://github.dev/PyCQA/pylint/blob/edc79e5392b0abfed222631501905ecd7521304e/doc/user_guide/checkers/features.rst#L520-L522)